### PR TITLE
Add overlay files for A22 5G and A34

### PR DESCRIPTION
**Pull request by Fede2782**
Add overlay files for A34 and A22 5G. Tested on my devices

### CODE REVIEW

File: Samsung/A34/res/xml/power_profile.xml [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
+<device name="Android">
+    <item name="ambient.on">110.69</item>
+    <item name="screen.on">91.6</item>
+    <item name="screen.full">230.56</item> public void onItem(Locator locator, String name, float value) throws SAXParseException {<item name="bluetooth.active">22.75</item>
```
Comment (phhusson) [inline Samsung/A34/res/xml/power_profile.xml]: copy-paste fail here.

ambient.on consuming more than screen.on sounds extremely unlikely to me

File: Samsung/A34/res/values/arrays.xml [hunk 2]
```diff
@@ -0,0 +1,1126 @@
...
+        <item>5000</item>
+        <item>6000</item>
+        <item>7000</item>
+        <item>10000</item>
+        <item>20000</item>
+        <item>50000</item>
+        <item>999999</item>
+    </integer-array>
+    <string-array name="config_defaultPinnerServiceFiles">
+        <item>/system/framework/arm/boot-mediatek-framework.vdex</item>
```
Comment (phhusson) [inline Samsung/A34/res/values/arrays.xml]: This file doesn't exist in a GSI :shrug:

**Comment by phhusson** [comment]
What does it fix? What precise features have been actually tested?

**Comment by Fede2782** [comment]
I'll try to fix these. I found everything in the overlay apk in vendor. I didn't modify anything. I also thought there was something wrong in power profile. Sorry for the broken commit.

**Pull request closed without merging**